documentation/assemblies/deploying/assembly-rolling-updates.adoc

```diff
@@ -3,9 +3,9 @@
 // using/assembly-management-tasks.adoc
 
 [id='assembly-rolling-updates-{context}']
-= Starting rolling updates of Kafka and other operands using annotations
-
-Strimzi supports the use of annotations to manually trigger a rolling update of Kafka and other operands through the Cluster Operator.
+= Managing rolling updates 
+Maintenance time windows allow you to schedule rolling updates of your Kafka and ZooKeeper clusters to start at a convenient time.
```

Comment:
I think this is misleading as it applies only to the rolling updates triggering completely automatically by the operator - which is basically only certificate renewal. All other rolling udpates are tirggered by user actiona and do not use the maintenance window. So I would remove it from here or explain it in detail.

Reply:
I did think about moving it to sit with the content on certificate renewal and validity periods, but I thought it might be a bit restrictive. I've moved now and updated some of the content in the section around it. (I think the TLS chapter in general could be refactored a bit more, but that's for another time)